### ISSUE TITLE
[bug] Console errors when used behind nginx reverse proxy?

### ISSUE THREAD

**Issue by YouveGotMeowxy** [mickael-kerjean/filestash#458]
I'm using the SWAG Nginx container and a reverse proxy with it. For the most part I have it working (except for dropbox auth), but notice this error a lot in my console. Does anyone know how to resolve this?

>The resource https://filestash.my.domain/api/config was preloaded using link preload but not used within a few seconds from the window's load event. Please make sure it has an appropriate `as` value and it is preloaded intentionally.

My current RP config looks like this:

```
server {
    listen 443 ssl;
    listen [::]:443 ssl;

    server_name filestash.*;

    include /config/nginx/ssl.conf;

    client_max_body_size 0;

    location / {

        include /config/nginx/proxy.conf;
        include /config/nginx/resolver.conf;
        set $upstream_app filestash;
        set $upstream_port 8334;
        set $upstream_proto http;
        proxy_pass $upstream_proto://$upstream_app:$upstream_port;

        # Added for compatibility w/Cloudflare.
        add_header Content-Security-Policy "script-src 'self' 'unsafe-inline';";
    }
    
    location ~ (/filestash)?/api/session/auth/dropbox {
        include /config/nginx/proxy.conf;
        include /config/nginx/resolver.conf;
        set $upstream_app filestash;
        set $upstream_port 8334;
        set $upstream_proto http;
        proxy_pass $upstream_proto://$upstream_app:$upstream_port;
        add_header Content-Security-Policy "script-src 'self' 'unsafe-inline';";   
    }    
}

```

side note, my dropbox error when trying to auth sends me to an "oops, not allowed" page, and:

> GET https://filestash.my.domain/api/session/auth/dropbox 403

**Comment by mickael-kerjean** [comment]
That's a new and weird one. That endpoint is used to configure the frontend to reflect the configuration you've set in the admin console. I tried to preload it as we need always need it and use it, not sure why chrome would say otherwise

**Comment by YouveGotMeowxy** [comment]
> That's a new and weird one. That endpoint is used to configure the frontend to reflect the configuration you've set in the admin console. I tried to preload it as it makes the boot time slightly faster as we do always need it and use it, not sure why chrome would say otherwise, to be investigated

Could it have to do with the fact it's behind a reverse proxy? And/or the CSP header I added?

**Comment by mickael-kerjean** [comment]
You don't need to add any CSP, Filestash already handles that stuff in its code

**Comment by YouveGotMeowxy** [comment]
> You don't need to add any CSP, Filestash already handles that stuff in its code

ok, I usually add that line if I suspect a problem with cloudflares extra scripts, like rocketloader. I did disable that in the RP config and no issues with FS. :)

I also got the dropbox 'issue' fixed (I was accidentally using the super long app key instead if the short ID key. ope!

Another 'issue' I fixed was due to accidentally adding the `https://` with my URL to the `host:` field; removing it solved my issue (I think I was having problems  using the logout button when I had the schema added, but I forget now after wrasslin with a bunch of various problems at once. Might be helpful for others to note not to add http/s in the help text underneath it.

I do still get those "preloaded" errors in the console though. I have no idea what I could do on my end to try to fix that.
